lib/licensed/dependency.rb

```diff
@@ -96,21 +96,28 @@ def remote_license_file
     end
```

Comment:
Not part of this change, but just idly wondering why above two lines can't just be

`@remote_license_file ||= Licensed.from_github(self["homepage"])`

Reply:
Good question!  It's because `||=` doesn't stop `Licensed.from_github(self["homepage"])` from being called again when a previous call returned `nil`